plugins/modules/change_request_task.py

```diff
@@ -0,0 +1,436 @@
+  hold_reason:
+    description:
+      - Reason why change task is on hold.
+      - Required if change task's I(on_hold) value will be C(true).
+    type: str
+  planned_start_date:
+    description:
+      - The date you plan to begin working on the task.
+    type: str
+  planned_end_date:
+    description:
+      - The date the change task is planned to be completed.
+    type: str
+  close_code:
+    description:
+      - Provide information on how the change task was resolved.
+      - The change task must have this parameter set prior to
+        transitioning to the C(closed) state.
+    choices: [ successful, successful_issues, unsuccessful ]
+    type: str
+  close_notes:
+    description:
+      - Resolution notes added by the user who closed the change task.
+      - The change task must have this parameter set prior to
+        transitioning to the C(closed) state.
+    type: str
+  other:
+    description:
+      - Optional remaining parameters.
+      - For more information on optional parameters, refer to the ServiceNow
+        change task documentation at
+        U(https://docs.servicenow.com/bundle/paris-it-service-management/page/product/change-management/task/create-a-change-task.html).
+    type: dict
+"""
+
+EXAMPLES = """
+- name: Create a change task
+  servicenow.itsm.change_request_task:
+    configuration_item: Rogue Squadron Launcher
+    change_request_number: CHG0000001
+    type: planning
+    state: open
+    assigned_to: fred.luddy
+    assignment_group: robot.embedded
+    short_description: Implement collision avoidance
+    description: "Implement collision avoidance based on the newly installed TOF sensor arrays."
+    on_hold: true
+    hold_reason: "Waiting for a report from the hardware team"
+    planned_start_date: 2021-07-15 08:00:00
+    planned_end_date: 2021-07-21 16:00:00
+    other:
+      approval: approved
+  
+- name: Change state of the change task
+  servicenow.itsm.change_request_task:
+    state: in_progress
+    on_hold: false
+    number: CTASK0000001
+
+- name: Close a change task
+  servicenow.itsm.change_request_task:
+    state: closed
+    close_code: "successful"
+    close_notes: "Closed"
+    number: CTASK0000001
+
+- name: Delete a change task
+  servicenow.itsm.change_request_task:
+    state: absent
+    number: CTASK0000001
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ..module_utils import arguments, client, table, errors, utils, validation
+from ..module_utils.change_request_task import PAYLOAD_FIELDS_MAPPING
+
+DIRECT_PAYLOAD_FIELDS = (
+    "state",
+    "short_description",
+    "description",
+    "on_hold",
+    "planned_start_date",
+    "planned_end_date",
+    "close_code",
+    "close_notes",
+)
+
+
+def ensure_absent(module, table_client):
+    mapper = utils.PayloadMapper(PAYLOAD_FIELDS_MAPPING, module.warn)
+    query = utils.filter_dict(module.params, "sys_id", "number")
+    change = table_client.get_record("change_task", query)
```

Comment:
```suggestion
    task = table_client.get_record("change_task", query)
```

Or `change_task`, whatever feels more natural. But `change` is not OK because it is misleading.